subprojects/docs/src/snippets/plugins/buildscript/kotlin/build.gradle.kts

```diff
@@ -4,9 +4,10 @@ buildscript {
         gradlePluginPortal()
     }
     dependencies {
-        classpath("com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.5")
-    }
+        classpath(id("com.jfrog.bintray", "1.8.5")) // <1>
```

Comment:
:question: What's the `id()` notation here?

Reply:
An error - I updated the snippet.